src/Symfony/Bridge/PhpUnit/composer.json

```diff
@@ -18,10 +18,10 @@
     "require": {
         "php": ">=7.1.3 EVEN ON LATEST SYMFONY VERSIONS TO ALLOW USING",
         "php": "THIS BRIDGE WHEN TESTING LOWEST SYMFONY VERSIONS.",
-        "php": ">=7.1.3",
-        "symfony/deprecation-contracts": "^2.1"
+        "php": ">=7.1.3"
     },
     "require-dev": {
+        "symfony/deprecation-contracts": "^2.1",
```

Comment:
hmmm whats for?

Reply:
`trigger_deprecation` is still used in tests of this component.